### ISSUE TITLE
Valid Pixabay API Key not accepted

### ISSUE THREAD

**Issue by aloscher** [vanjarosoftware/Vanjaro.Platform#1624]
**Product & Version**
Vanjaro for DNN 

**Describe the bug**
Valid Pixabay API Key not accepted - message "Input is not a valid Pixabay API Key."

**To Reproduce**
Menu->Settings->Integrations & Connectors->Pixabay->Global or Site

**Expected behavior**
API Implemented

**Current behavior**

 "Input is not a valid Pixabay API Key."

**Screenshots**

**Additional context**
Windows 10 - same results in Firefox Chrome and Edge browsers

**Comment by Tychodewaard** [comment]
I use Vanjaro (1.1) for DNN (9.9.1) and at yesterday I had a working Pixabay.
But...today images became 404. The url was not valid. If I open up the connector, there is an empty text box.
The Youtube API text box is also empty.

This a site as one of 6 portals and I was using the API on portal level.

**Comment by nikhilmandeeps** [comment]
> **Product & Version**
> Vanjaro for DNN
> 
> **Describe the bug**
> Valid Pixabay API Key not accepted - message "Input is not a valid Pixabay API Key."
> 
> **To Reproduce**
> Steps to reproduce the behavior:
> Menu->Settings->Integrations & Connectors->Pixabay->Global or Site
> 
> **Expected behavior**
> API Implemented
> 
> **Current behavior**
> 
> "Input is not a valid Pixabay API Key."
> 
> **Screenshots**
> If applicable, add screenshots to help explain your problem.
> 
> **Additional context**
> Windows 10 - same results in Firefox Chrome and Edge browsers


Can you please provide a screencast

**Comment by meetmandeep** [comment]
> But...today images became 404.

This should never happen. We don't hotlink images from PixaBay. They're downloaded and served from your site filesystem. It's likely you have hot-linked images; if that's the case, please open a new issue with steps to duplicate. 

> If I open up the connector, there is an empty text box.
> The YouTube API text box is also empty.

Can you provide a screencast? It's possible that you configured the setting at Global Level (logged in as host) and viewing it at site level (logged in as administrator)?

**Comment by scottw12** [comment]
Actually the problem goes deeper. i have a brand new Platform install with nothing done yet and I cannot even register integrations or connectors except Azure.. See attached video;; http://somup.com/crjFqmYRJ7

**Comment by meetmandeep** [comment]
@scottw12 Are there any errors in console?

**Comment by scottw12** [comment]
No, I watching  when I went to the virgin site.  However after I sent eh video I restarted the application and now things are working fine for me and pixabay and YouTube are working on three other sites including one that is Vanjaro_for_DNN.

I noticed that all of the items that did not work when I did the video are ones that open up in  a modal.  I had problems with Live_Content with modals on Vanjaro and it was fixed in the Live_Content module.

**Comment by scottw12** [comment]
I put my Pixaby API Key into the virgin installed Platform and it is working fine. https://vanjaro.jkmcomm.com/
I did it both ways as Site and then removed and added as Global. Also working for in Vanjaro_for_DNN   https://dnn.jkmcomm.com/tst

**Comment by aloscher** [comment]
> 
> 
> > **Product & Version**
> > Vanjaro for DNN
> > **Describe the bug**
> > Valid Pixabay API Key not accepted - message "Input is not a valid Pixabay API Key."
> > **To Reproduce**
> > Steps to reproduce the behavior:
> > Menu->Settings->Integrations & Connectors->Pixabay->Global or Site
> > **Expected behavior**
> > API Implemented
> > **Current behavior**
> > "Input is not a valid Pixabay API Key."
> > **Screenshots**
> > If applicable, add screenshots to help explain your problem.
> > **Additional context**
> > Windows 10 - same results in Firefox Chrome and Edge browsers
> 
> Can you please provide a screencast


https://user-images.githubusercontent.com/21082075/130086855-9fd044ea-4836-4627-96be-0e37fccd9bd7.mp4

![sitelog](https://user-images.githubusercontent.com/21082075/130086919-479f07c5-2e4e-41d3-a220-00fa9029ef7e.png)

**Comment by mksmanish79** [comment]
https://pixabay.com/api/?key=yourkey

Check if this url is giving you json response by replacing yourkey with your key.

**Comment by mksmanish79** [comment]
Also provide us the screenshot of the the error logged by clicking on the exception. You have not given the full screenshot of the error logged.

**Comment by aloscher** [comment]
> 
> 
> Also provide us the screenshot of the the error logged by clicking on the exception. You have not given the full screenshot of the error logged.

![image](https://user-images.githubusercontent.com/21082075/130220176-dfe4777d-9769-4c53-9f99-4a1e76d60db6.png)

**Comment by aloscher** [comment]
> 
> 
> https://pixabay.com/api/?key=yourkey
> 
> Check if this url is giving you json response by replacing yourkey with your key.

I received a successful reply from the url

**Comment by mksmanish79** [comment]
Can you please check after disabling your antivirus/firewall for a while?

**Comment by aloscher** [comment]
> 
> 
> Can you please check after disabling your antivirus/firewall for a while?

I just tried without my antivirus/firewall with the same results. I even tried another browser.

**Comment by meetmandeep** [comment]
@aloscher The error log provided indicates that your server is unable to connect to Pixabay; I suspect firewall as an issue. Please review your outbound connections from server.

**Comment by aloscher** [comment]
> 
> 
> @aloscher The error log provided indicates that your server is unable to connect to Pixabay; I suspect firewall as an issue. Please review your outbound connections from server.

I will check with my host.
